src/server/emulatorStaticServer.ts

```diff
@@ -0,0 +1,41 @@
+import { readFileSync } from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import cors from "cors";
+import express, { type RequestHandler } from "express";
+
+/**
+ * Serve the built emulator React app
+ * This router serves static files from the emulator's dist directory.
+ * It should be installed at the application root, like so:
+ *
+ *  const app = express();
+ *
+ * if (env.NODE_ENV !== "production") {
+ *   app.use(await emulatorStaticServer(server));
```

Comment:
the doc says you should have the server as param but not the actual implementation